src/database/src/config.rs

```diff
@@ -0,0 +1,3 @@
+pub const NUM_INSTANCES: u64 = 20;
```

Comment:
Is the current model 20 regions, 1 partition per region?

Reply:
Yep